### PR TITLE
using MEMBRANE_PRESTRESS

### PR DESCRIPTION
**Pull request by KlausBSautter**
with this PR the membrane element uses the MEMBRANE_PRESTRESS if it is defined

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp [hunk 1]
```diff
@@ -1157,21 +1173,20 @@ void MembraneElement::CalculateAndAddBodyForce(VectorType& rRightHandSideVector)
 {
     KRATOS_TRY
     auto& r_geom = GetGeometry();
-    if (r_geom[0].SolutionStepsDataHas(VOLUME_ACCELERATION)){
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp]: why do you remove this?

Comment (KlausBSautter) [reply applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp]: this was not intended .... sry :flight_arrival:

File: applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp [hunk 2]
```diff
@@ -975,6 +980,17 @@ void MembraneElement::Calculate(const Variable<Matrix>& rVariable, Matrix& rOutp
         column(rOutput,1) = base_2;
         column(rOutput,2) = base_3;
     }
+    else if (rVariable == MEMBRANE_PRESTRESS) {
+        std::vector< Vector > prestress_matrix;
+        CalculateOnIntegrationPoints(PK2_STRESS_VECTOR,prestress_matrix,rCurrentProcessInfo);
+        const GeometryType::IntegrationPointsArrayType& r_integration_points = GetGeometry().IntegrationPoints(GetGeometry().GetDefaultIntegrationMethod());
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp]:
```suggestion
        const auto& r_integration_points = GetGeometry().IntegrationPoints(GetGeometry().GetDefaultIntegrationMethod());
```

**Review by philbucher — COMMENTED** [review]
minor otherwise ok

**Review by philbucher — APPROVED** [review]

**Pull request merged**